frontend/www/js_templates/l2/endpoint_selection_modal.js

```diff
@@ -325,22 +325,38 @@ class EndpointSelectionModal2 {
         disabled = 'disabled';
         notAllow = 'cursor: not-allowed;';
       }
-
-      let elem = document.createElement('li');
+      var elem = document.createElement('li');
```

Comment:
Unless there's a reason for using `var`, `let` is preferred as it limits the variables scope.